Formula/mu.rb

```diff
@@ -53,13 +49,15 @@ class Mu < Formula
   fails_with gcc: "5"
 
   def install
-    system "autoreconf", "-ivf" if build.head?
-    system "./configure", "--disable-dependency-tracking",
-                          "--disable-guile",
-                          "--prefix=#{prefix}",
-                          "--with-lispdir=#{elisp}"
-    system "make"
-    system "make", "install"
+    mkdir "build" do
+      system "meson", *std_meson_args, ".."
+      system "ninja", "-v"
+      system "ninja", "install", "-v"
+    end
+
+    # fix lisp file install location
+    mkdir_p "#{share}/emacs/site-lisp/mu"
+    mv "#{share}/emacs/site-lisp/mu4e", "#{share}/emacs/site-lisp/mu/mu4e"
```

Comment:
any upstream issue tracker for this?

Reply:
Not sure, this updates the structure to the way it was in previous versions of `mu`.

Reply:
Created an issue: https://github.com/djcb/mu/issues/2287